src/astro/databases/google/bigquery.py

```diff
@@ -255,17 +257,23 @@ def load_gs_file_to_bigquery(
             "load": load_job_config,
             "labels": {"target_table": target_table.name},
         }
-        self.hook.insert_job(
-            configuration=job_config,
-        )
+        try:
+            self.hook.insert_job(
+                configuration=job_config,
+            )
+        # Ignoring deepsource error as it needs to catch every other exception
+        except Exception as exe:  # skipcq: PYL-W0703
```

Comment:
@sunank200 as per our discussion the other day, we need to handle errors in a way that if there is a config issue we retry with the default path, and if there is a network issue we can fail directly and not try the default path.

We can take this as a separate task since we need to explore what exception an underline lib can raise and then figure out where it is suitable to fail or retry with the default path.

Reply:
Thanks, @bharanidharan14 for pointing it out.

Reply:
Sure. I will add a separate issue and PR for this.

Reply:
Added the change in this [commit](https://github.com/astronomer/astro-sdk/pull/560/commits/48fa4e8145fb54db5ac3f295e3af70d1742c71ff)